src/style.css

```diff
@@ -1,3 +1,80 @@
 body {
+  width: 100%;
   margin: 0;
+  font-weight: 500;
+  font-family: Roboto, sans-serif;
+  font-size: 12px;
+  font-style: normal;
+  background: #fff;
+}
+
+ul {
+  list-style-type: none;
+}
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+}
+
+.logo-img {
+  width: 40px;
+  height: 40px;
+  display: block;
+}
+
+.nav-menu {
+  text-align: center;
+  line-height: 60px;
+}
+
+.menu-list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+  list-style: none;
+
+  /* gap: 20px;
+  padding-top: 22px;
+  padding-bottom: 22px; */
```

Comment:
This section of the '.menu-list' class is commented out. If these styles are intended to be part of the design, consider uncommenting them. Otherwise, if they are not needed, you can remove them to keep the code clean.